### DISCUSSION TITLE
Payment Uzbekistan card

### DISCUSSION THREAD

**Discussion by lostlang** [community/community#27835] [Copilot Conversations]
Hi, i’m student from Uzbekistan. I am a citizen of Uzbekistan. I’m living in Russia because i studies  in university. I wish buy and use Copilot, but i received messages “  After reviewing we have determined that your account is ineligible for GitHub purchases due to restrictions under U.S. economic sanctions.” . Uzbekistan people != russian people, i have Uzbekistan card and pasport, help me buy copilot please…..

P.S. Even payment in incentive works in Uzbekistan

**Comment by MrSarno** [comment]
Hello,

We will not be able to assist you with this, unfortunately, as it is working as intended.

Sanctions target **residents** of a country, not those who happen to have been born there. If you are living in Russia, you are subject to them. If you leave Russia and live elsewhere, you will cease being subject to them and you wiki be legally able to pay for Copilot.

**Reply by lostlang** [reply]
Nice position 🤡

**Reply by MrSarno** [reply]
I’m not sure why you feel the need to resort to sarcasm & an insulting emoji. I didn’t decide how it works.

**Reply by lostlang** [reply]
When the service punishes everyone, even non-residents (although not all residents are to blame for something) this is an "excellent" position of the service. My sarcasm is quite justified
